neurobooth_os/tasks/task.py

```diff
@@ -25,7 +26,22 @@
 from neurobooth_os.log_manager import APP_LOG_NAME
 import neurobooth_os.config as cfg
 
-from enum import Enum
+from pylsl import local_clock
+
+
+class TaskAborted(Exception):
+    """Exception raised when the task is aborted."""
+    pass
+
+
+def check_if_aborted(keys=("q",)) -> None:
+    """
+    Check to see if a task has been aborted. If so, raise an exception.
+    :param keys: The keys that will abort a task.
+    """
+    if event.getKeys(keyList=keys):
+        print(f"Task Aborted.")  # Send message to CTR
+        raise TaskAborted()
```

Comment:
did you test that the print statement shows up in CTR? As I understand it, this depends on the calling context.  If you tested it, you can ignore this.

Reply:
Will test; based on when I did something similar with MOT it should go to CTR.